apps/backend/api/models/Group.js

```diff
@@ -97,6 +97,10 @@ module.exports = bookshelf.Model.extend(merge({
     })
   },
 
+  contextWidgets () {
```

Comment:
I think I would call this contextMenuWidgets - the menu part feels important/helpful

Reply:
This returns all context widgets for the group, including ones not currently in the menu. So adding menu to the title might be slightly misleading